packages/vue-cli-plugin-vuetify/generator/templates/v3/src/plugins/vuetify.js

```diff
@@ -1,8 +1,8 @@
 import '@mdi/font/css/materialdesignicons.css'
-import 'vuetify/lib/styles/main.sass'
+import 'vuetify/styles'
 import { createVuetify } from 'vuetify'
-import * as components from 'vuetify/lib/components'
-import * as directives from 'vuetify/lib/directives'
+import * as components from 'vuetify/components'
+import * as directives from 'vuetify/directives'
```

Comment:
These are imported automatically now.

Reply:
I'm missing something somewhere then. When I try to generate a project with those imports removed, the components fail to resolve.